README.md

```diff
@@ -37,6 +37,19 @@ then flash the resulting image eg. to a SD card or USB drive
     dd if=./result/nixos.img of=/dev/<YOUR_USB_DRIVE> bs=32M
 
 
+### Cross compilation complications
+
+As there are some packages that are not cross-compilation aware you may set the following in you configuration.nix to enable binfmt.
```

Comment:
As there are some packages that are not cross-compilation aware **,** you **can** set the following in you `configuration.nix` to enable binfmt.